src/schedulerDependencies.js

```diff
@@ -0,0 +1,56 @@
+require('dotenv').config();
+const requestPromise = require('request-promise');
+const sgMail = require('@sendgrid/mail');
+
+/**
+ * @description Updates config variables on heroku
+ *
+ * @param {number} token takes the token to be updated on heroku
+ * @returns {void}
+ */
+const updateConfigOnHeroku = async (token) => {
+  try {
+    const options = {
+      method: 'PATCH',
+      url: `https://api.heroku.com/apps/${process.env.HEROKU_APP_NAME}/config-vars`,
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/vnd.heroku+json; version=3',
+        Authorization: `Bearer ${process.env.HEROKU_ACCESS_TOKEN}`
+      },
+      body: {
+        OAUTH_ACCESS_TOKEN: `${token}`
+      },
+      json: true
+    };
+    await requestPromise(options);
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+/**
+ * @description sends email to notify new token status
+ *
+ * @param {array} emails takes the token to be updated on heroku
+ * @param {number} token takes the new token
+ * @returns {void}
+ */
+const sendTokenNotificationEmail = (emails, token) => {
+  emails.forEach((email) => {
+    sgMail.send({
+      to: `${email}`,
+      from: 'no-reply@helloroo.org',
+      subject: 'Token Renewal Notification',
+      text: `The auth0 token has been renewed. Please update the token in 1password and the env file
+      Here is the new token is which expires after 100 days.
```

Comment:
Thanks for this great work. Please proofread the text on line 46.

Reply:
Thank you for that